plugin.video.mediathek/html.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#-------------LicenseHeader--------------
+# plugin.video.mediathek - Gives access to most video-platforms from German public service broadcasters
+# Copyright (C) 2010  Raptor 2101 [raptor2101@gmx.de]
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+
+def transformHtmlCodes(string):
+  replacements = (
+    (u'Ä', u'&Auml;'),
```

Comment:
Known, this lines comes from times of the plugin where this wasn't available or known to me ;)